File: front-end/src/app/shared/models/dto/team/create-team-dto.ts
import { TeamInvite, TeamMember } from './team-dto';

export interface CreateTeamDto {
  /**
   * The team's photo url
   */
  teamPhotoUrl: string;
  name: string;
  /**
   * The members of the team
   */
  members: TeamMember[];
  /**
   * The invites to the team
   */
  invites: TeamInvite[];
}

File: API/Application/DTO/CreateTeamDto.cs
using API.Core.Models;

namespace API.Application.DTO
{
	public class CreateTeamDto
	{
		/// <summary>
		/// The team's photo url
		/// </summary>
		public string TeamPhotoUrl { get; set; }
		public string Name { get; set; }
		/// <summary>
		/// The members of the team
		/// </summary>
		public List<TeamMember> Members { get; set; }
		/// <summary>
		/// The invites to the team
		/// </summary>
		public List<TeamInvite> Invites { get; set; }
	}
}
